Extensions/Firebase/JsExtension.js

```diff
@@ -321,6 +321,7 @@ module.exports = {
         'JsPlatform/Extensions/firebase.png',
         'JsPlatform/Extensions/firebase.png'
       )
+      .addParameter('string', _('Provider'), '', false)
```

Comment:
Good catch! I recommend using `stringWithSelector`:
```
.addParameter(
        'stringWithSelector',
        _('Provider'),
        '["google", "facebook", "github", "twitter"]',
        false
      )
```

Reply:
Ah, nice, I looked for this but couldn't find it in any docs or other examples :D